custom_components/opensprinkler/__init__.py

```diff
@@ -177,94 +177,38 @@ def device_state_attributes(self):
         controller = self._controller
         attributes = {"opensprinkler_type": "controller"}
         for attr in [
-            "enabled",
             "firmware_version",
             "hardware_version",
             "hardware_type",
-            "hardware_type_name",
```

Comment:
I removed a lot of the `*_name` ones as I felt that for automations it better to use the non "friendly" types. Thoughts?